src/app/iteration/iteration.component.html

```diff
@@ -11,17 +11,16 @@
         </h3>
       </div>
       <ul class="iteration-expanded-div" [collapse]="isCollapsedCurrentIteration">
-        <li class="active" *ngFor="let iteration of currentIterations" [routerLinkActive]="'selected-iteration'">
+        <li class="active" *ngFor="let iteration of currentIterations" [routerLinkActive]="'selected-iteration'" 
```

Comment:
Uhh..We should *not* support open in new tabs anyway at this point. This causes all sorts of problems, lost updates are the least. So I think explicitly disabling that is actually a good thing.